myria/cluster/scripts/cli.py

```diff
@@ -1141,7 +1141,20 @@ def run():
     help="Number of Myria workers per cluster node [default: %d]" % INSTANCE_TYPE_CONFIGS[DEFAULTS['instance_type']].workers_per_node)
 @click.option('--cluster-log-level', show_default=True,
     type=click.Choice(LOG_LEVELS), default=DEFAULTS['cluster_log_level'])
+@click.option('--perfenforce', is_flag=True, help="Enable PerfEnforce (will override default cluster configuration)")
 def create_cluster(cluster_name, **kwargs):
+    # If perfenforce is enabled, override the cluster configuration
+    if kwargs['perfenforce']:
+        click.echo("Adjusting cluster options for PerfEnforce...")
+        kwargs['cluster_size'] = 13
+        kwargs['instance_type'] = 'm4.xlarge'
+        kwargs['worker_mem'] = 12
+        kwargs['worker_vcores'] = 2
+        kwargs['node_mem_gb'] = 13
+        kwargs['node_vcores'] = 3
+        kwargs['workers_per_node'] = 1
+        kwargs['unprovisioned'] = True
```

Comment:
Assuming this will be removed once all the pending features are integrated into RACO/myria `master`?